### ISSUE TITLE
Summary Widget

### ISSUE THREAD

**Issue by thomprl** [SIMRacingApps/SIMRacingApps#223]
Maybe have a summary widget to show while under caution maybe just the first lap or 2 that shows the track name, number of cautions, caution laps, current weather, lead changes, last lap pit stop, avg speed of the race, current length of the race in time ,laps to go.  Just something to break up the race and give streamers something to show.

**Comment by jfgilliam** [comment]
New API's needed for this:

/Session/LeadChanges
/Session/AverageSpeed

As for when it shows, I could hide it until a caution comes out. But some may want to see this all the time. From a streaming standpoint, is there a way a "Scene" can be triggered by a URL value? What I mean is create a Caution Scene, then have it activate based on the race status.

**Comment by thomprl** [comment]
Yeah I don't know of a way to do that, but I'll look into it.  I know there is a OBS Remote.
